material-flux/package.json

```diff
@@ -9,7 +9,7 @@
       [
         "babelify",
         {
-          "experimental": true,
+          "stage": 1,
```

Comment:
I'm not sure if material-flux actually needs this, nothing seemed to fail without it so it probably doesn't use any stage 0 or 1 features.